pkg/template/summary_printer.go

```diff
@@ -0,0 +1,194 @@
+package template
+
+import (
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/influxdata/influx-cli/v2/api"
+	"github.com/influxdata/influx-cli/v2/pkg/influxid"
+)
+
+func PrintSummary(summary api.TemplateSummaryResources, out io.Writer, useColor bool, useBorders bool) error {
+
+	newPrinter := func(title string, headers []string) *TablePrinter {
+		return NewTablePrinter(out, useColor, useBorders).
+			Title(title).
+			SetHeaders(append([]string{"Package Name", "ID", "Resource Name"}, headers...)...)
+	}
+
+	if labels := summary.Labels; len(labels) > 0 {
```

Comment:
The thought has crossed my mind a few times while looking at code related to all this pkger stuff that "this sure is a lot of things that are almost the same but not quite".  To be sure, considering this is mostly a port of working code I don't think there is any reason to try to DRY it up right now, but if somebody gets bored one day...

Reply:
It's really begging for generics